rotkehlchen/api/v1/schemas.py

```diff
@@ -1019,33 +1019,22 @@ def make_history_base_entry(self, data: dict[str, Any], **_kwargs: Any) -> dict[
             return {'events': events}
 
     class CreateEvmSwapEventSchema(BaseEvmEventSchema):
+
+        class EvmSwapSubEventSchema(Schema):
+            identifier = fields.Integer(required=False, load_default=None)
+            amount = AmountField(validate=validate.Range(min=ZERO, min_inclusive=False))
```

Comment:
```suggestion
            amount = AmountField(validate=validate.Range(min=ZERO, min_inclusive=False), required=True)
```

don't you need to set required=True here?

Reply:
ah, yes. Changed this. Also I realized that I hadn't mentioned the identifier field in the api docs so added that as well.